src/curl.cpp

```diff
@@ -106,26 +104,26 @@ sse_type_t       S3fsCurl::ssetype             = sse_type_t::SSE_DISABLE;
 bool             S3fsCurl::is_content_md5      = false;
 bool             S3fsCurl::is_verbose          = false;
 bool             S3fsCurl::is_dump_body        = false;
-string           S3fsCurl::AWSAccessKeyId;
-string           S3fsCurl::AWSSecretAccessKey;
-string           S3fsCurl::AWSAccessToken;
+std::string           S3fsCurl::AWSAccessKeyId;
```

Comment:
If possible, please align the indentation.

Reply:
Done.